src/inline_tests.ml

```diff
@@ -233,6 +233,7 @@ include Sub_system.Register_end_point(
           ~scope
           ~dir:inline_test_dir
           ~modules
+          ~opaque:true
```

Comment:
Not entirely sure that this is correct, but it seems like we'd want the tests to compiler as fast as possible. @diml?

Reply:
It's not going to make much difference as this context is only used to build the test runner which contains the entry point and so is not used by anything else. Given that we should stick to the default, i.e. `opaque=false`.